### PR TITLE
[testscripts] respect DEVBOX_DEBUG flag instead of always setting it on

### PR DESCRIPTION
**Pull request by savil**
## Summary

It is very hard to parse the output from testscripts because we always turn on DEVBOX_DEBUG output as well.

Instead, we should respect the invoking user's DEVBOX_DEBUG value.

## How was it tested?

shows output as before:
```
 DEVBOX_DEBUG=1 go test -v ./testscripts/...
```

cleaner output:
```
go test -v ./testscripts/...
```

### CODE REVIEW

**Comment by savil** [comment]
Current dependencies on/for this PR:
* main
  * **PR #824** <a href="https://app.graphite.dev/github/pr/jetpack-io/devbox/824" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jetpack-io/devbox/824?utm_source=stack-comment).

**Review by gcurtis — APPROVED** [review]
